docs/useMediaQuery.md

```diff
@@ -27,13 +27,13 @@ const isSmall = useMediaQuery('(min-width:600px)');
 
 ## Responsive Layouts
 
-Here is an example for a responsive list of posts, displaying a `SimpleList` on mobile, and a `Datagrid` otherwise:
+Here is an example for a responsive list of posts, displaying a `SimpleList` on mobile, and a `DataTable` otherwise:
```

Comment:
```suggestion
Here is an example for a responsive list of posts, displaying a [`<SimpleList>`](./SimpleList.md) on mobile, and a [`<DataTable>`](./DataTable.md) otherwise:
```